### DISCUSSION TITLE
How to use multiple subfolder for differente topics to create a website with quarto on github ?

### DISCUSSION THREAD

**Discussion by Fabelab** [quarto-dev/quarto-cli#9769] [Q&A]
### Description

I'm publishing a website on github, how could I group related qmd files on each topics in their own respective subfolder for organization sake to be render normally. Are they some setting to use inside _quarto.yml

Using windows 10 quarto 1.4.553 with Vscode

**Comment by mcanouil** [comment]
Did you look at the listing feature? https://quarto.org/docs/websites/website-listings.html

**Comment by Fabelab** [comment]
I must say that is a all new perspective to the topic, it is very interesting and very compelling feature, I can already see a bunch of cases where it could perfectly fit, although I would try just to see how it handle for example the rendering order of each added docs, I see there is option to change sorting parameters ...
Any way I found the solution early in the morning, to my own question, I search the net and even asked chatgpt help to figure out the cleanest option, but all failed, dealing with multiple index.qmd in root and subfolders ... i just created a folder and put *.qmd there I left on root index.qmd. about.qmd, etc... and all config files and all is working as a charm! 

The best part is that it would have been what I wanted to do in the first place instinctively, but I thought It can't be so simple and without dare consequences, let's check online lol

but thanks to your insight all was worth the dedicated time!
Thx again

**Reply by mcanouil** [reply]
FYI, you could take a more in depth look at the website guide in the documentation and look at the website gallery in the documentation and in the GitHub Awesome List.

- https://quarto.org/docs/websites/
- https://github.com/mcanouil/awesome-quarto

Note that you should consider reading and looking in documentations before relying on LLM.
